src/Exams/Exam2/Exam2.js

```diff
@@ -122,6 +123,62 @@ export const Exam2 = () => {
       question: 'Чему равен z?',
       answer: /['"]?100px3.14['"]?/,
     },
+    {
+      code: `
+      let x = 100;
+      let y = x + 'px';
+      let w = y + x - y * 2;
```

Comment:
I think it's `NaN` 

![image](https://user-images.githubusercontent.com/6686679/231242915-633c3020-8841-4706-abf3-bde7c3d55655.png)

Reply:
confused himself. I want ask z variable